.pfnci/common.sh

```diff
@@ -12,6 +12,9 @@ gcloud auth configure-docker
 TEMP=$(mktemp -d)
 mount -t tmpfs tmpfs ${TEMP}/ -o size=100%
 
+git merge-base --is-ancestor HEAD v6 && LOCAL_VERSION=stable
+git merge-base --is-ancestor HEAD master && LOCAL_VERSION=master
```

Comment:
FlexCI does not provide `.git`. So these `git` commands always fail.

Reply:
Hmm, I see. What can be used, then...?

Reply:
@kmaehashi Oh, that's nice! I put the setting into the configs.
@Hakuyume OK, I updated to avoid searching ancestors for chainercv case.

Reply:
Updating ChainerCV's `config.pbtxt` doesn't affect Chainer/CuPy's CI.
Please fix the following files instead.
https://github.com/chainer/chainer/blob/master/.pfnci/config.pbtxt
https://github.com/cupy/cupy/blob/master/.pfnci/config.pbtxt

Reply:
Oh, OK, I'll revert the change in this PR and fix each of Chainer/CuPy's config.